tools/validator-client-scripts/src/commands/migrate_contract.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::Client;
+use clap::Parser;
+use log::info;
+use validator_client::nymd::cosmwasm_client::types::{ContractCodeId, EmptyMsg};
+
+#[derive(Debug, Parser)]
+pub(crate) struct Args {
+    #[clap(long)]
+    pub code_id: ContractCodeId,
+
+    #[clap(long)]
+    pub memo: Option<String>,
+
+    #[clap(long)]
+    pub init_message: Option<String>,
+
+    #[clap(long)]
+    pub gas: Option<u64>,
+}
+
+pub(crate) async fn migrate(client: Client, args: Args) {
+    println!("Starting contract migration!");
+
+    let memo = args.memo.unwrap_or_else(|| "contract migration".to_owned());
+    let contract_address = client.mixnet_contract_address();
```

Comment:
We should support (at least) two configurable contract migrations: mixnet and vesting. Hardcoding it to mixnet here would limit that for us. I would say, pass the migrating contract address as an migration argument instead.